### ISSUE TITLE
Scroll to bottom button missing accessible name

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3474]
**Issue Description:** The button to scroll to the bottom of the chat has no accessible name and is only described as "button".

**WCAG Criteria:** [4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG22/#name-role-value)

**Screenshot or Snippet:** [Link to Screenshot](https://share.zight.com/P8urBkBZ)

**Notes:** All interactive elements and form fields should have a unique, descriptive name that identifies the element uniquely from other elements. This issue can usually be found with automated checks.
